push_notifications/apns.py

```diff
@@ -48,8 +48,8 @@ def _check_certificate(ss):
 		raise Exception("The certificate doesn't contain a private key")
 
 
-def _apns_create_socket(address_tuple):
-	certfile = SETTINGS.get("APNS_CERTIFICATE")
+def _apns_create_socket(address_tuple, certfile=None, ca_certs=None):
+	certfile = SETTINGS.get("APNS_CERTIFICATE") if not certfile else certfile
```

Comment:
would be simpler just to pass it as an arg to get
` SETTINGS.get("APNS_CERTIFICATE"), certificate`